ui/src/pages/insightDetail/components/podLogs/styles.module.less

```diff
@@ -469,256 +469,272 @@
-                  color: lighten(#9370db, 10%);
-                  border-bottom-color: currentColor;
+                  &:hover {
+                    color: lighten(#9370db, 10%);
+                    border-bottom-color: currentColor;
+                  }
                 }
-              }
 
-              table {
-                border-collapse: separate;
-                border-spacing: 0;
-                width: 100%;
-                margin: 16px 0;
-                border-radius: 8px;
-                border: 1px solid rgba(147, 112, 219, 0.2);
-                overflow: hidden;
-
-                th,
-                td {
+                table {
+                  border-collapse: separate;
+                  border-spacing: 0;
+                  width: 100%;
+                  margin: 16px 0;
+                  border-radius: 8px;
                   border: 1px solid rgba(147, 112, 219, 0.2);
-                  padding: 12px;
-                }
-
-                th {
-                  background: rgba(147, 112, 219, 0.1);
-                  font-weight: 600;
-                  text-align: left;
-                }
+                  overflow: hidden;
 
-                tr {
-                  background-color: transparent;
-                  transition: background-color 0.2s ease;
+                  th,
+                  td {
+                    border: 1px solid rgba(147, 112, 219, 0.2);
+                    padding: 12px;
+                  }
 
-                  &:nth-child(2n) {
-                    background-color: rgba(147, 112, 219, 0.05);
+                  th {
+                    background: rgba(147, 112, 219, 0.1);
+                    font-weight: 600;
+                    text-align: left;
                   }
 
-                  &:hover {
-                    background-color: rgba(147, 112, 219, 0.1);
+                  tr {
+                    background-color: transparent;
+                    transition: background-color 0.2s ease;
+
+                    &:nth-child(2n) {
+                      background-color: rgba(147, 112, 219, 0.05);
+                    }
+
+                    &:hover {
+                      background-color: rgba(147, 112, 219, 0.1);
+                    }
                   }
                 }
-              }
 
-              hr {
-                border: none;
-                height: 1px;
-                background: linear-gradient(to right,
-                    rgba(147, 112, 219, 0.1),
-                    rgba(147, 112, 219, 0.4),
-                    rgba(147, 112, 219, 0.1));
-                margin: 24px 0;
+                hr {
+                  border: none;
+                  height: 1px;
+                  background: linear-gradient(to right,
+                      rgba(147, 112, 219, 0.1),
+                      rgba(147, 112, 219, 0.4),
+                      rgba(147, 112, 219, 0.1));
+                  margin: 24px 0;
+                }
```

Comment:
The `.streamingIndicator` class has been moved inside `.diagnosisContent`, which improves the structure by grouping related elements.